docs/CONTRIBUTING.md

```diff
@@ -14,14 +14,6 @@ $ export GOPATH=$HOME/go
 $ export PATH=$PATH:$GOPATH/bin
 ```
 
-## Dependency management
-
-TiDB Operator uses [retool](https://github.com/twitchtv/retool) to manage Go related tools.
-
-```sh
-$ go get -u github.com/twitchtv/retool
-```
```

Comment:
`retool` will be installed when necessary (`make check-setup`), no need to install manually.